### PR TITLE
Scheduler failed payment

### PR DESCRIPTION
**Pull request by arkanoider**
Hi @grunch ,

putting here first draft to have a scheduled retry mechanism for failed ln payments:

- Added in settings.toml retry interval and number parameters.
- Added logic in `scheduler.rs` to mange retries.
- Modified `release.rs` to manage failed state of payment.

Putting as a draft here. 
Will upload also mostro-core with new `PaidHoldInvoice` state of the order to manage it in addinvoice.rs.

Give a look!

### CODE REVIEW

File: src/app/release.rs [hunk 1]
```diff
@@ -100,8 +100,9 @@ pub async fn do_payment(order: Order) -> Result<()> {
     // Finally we try to pay buyer's invoice
     let payment_request = order.buyer_invoice.as_ref().unwrap().to_string();
     let ln_addr = LightningAddress::from_str(&payment_request);
+    let amount = order.amount as u64 - order.fee as u64;
+    println!("Amount: {}", amount);
```
Comment (grunch) [inline src/app/release.rs:104]: I forgot to remove this on the patch file, can you remove this println ?

**Comment by grunch** [comment]
can you rebase this to main?

File: src/app/release.rs (hunk 1 above)
Comment (arkanoider) [reply src/app/release.rs:104]: Sure!

**Comment by grunch** [comment]
closing this one in favor of #206

**Pull request closed without merging**